edflow/main.py

```diff
@@ -186,6 +186,7 @@ def _test(config, root, nogpu=False, bar_position=0):
         bar_position=bar_position,
         nogpu=config["nogpu"],
         num_epochs=config["num_epochs"],
+        checkpoint_path=checkpoint,
```

Comment:
In the training case, we call the `initialize` method of the iterator with the checkpoint. I think we should handle the `checkpoint_path` consistently during training and evaluation. Just call initialize as in line 131?

Reply:
should be fixed

Reply:
cool. minor things: lines 176-178 should now be deleted. `checkpoint_path` is not required for the `compat_kwargs` anymore, but it also doesn't bother me being in there (but in `_train` the checkpoint path is not in the compat kwargs)